compiler-core/src/erlang/tests/snapshots/gleam_core__erlang__tests__variables__module_const_vars.snap

```diff
@@ -0,0 +1,27 @@
+---
+source: compiler-core/src/erlang/tests/variables.rs
+expression: "const int = 42\nconst int_alias = int\npub fn use_int_alias() { int_alias }\n\nfn int_identity(i: Int) { i }\nconst int_identity_alias: fn(Int) -> Int = int_identity\npub fn use_int_identity_alias() { int_identity_alias(42) }\n\nconst compound: #(Int, fn(Int) -> Int, fn(Int) -> Int) = #(int, int_identity, int_identity_alias)\npub fn use_compound() { compound.1(compound.0) }\n"
+---
+-module(the_app).
+-compile(no_auto_import).
+
+-export([use_int_alias/0, use_int_identity_alias/0, use_compound/0]).
+
+-spec use_int_alias() -> integer().
+use_int_alias() ->
+    42.
+
+-spec int_identity(integer()) -> integer().
+int_identity(I) ->
+    I.
+
+-spec use_int_identity_alias() -> integer().
+use_int_identity_alias() ->
+    fun int_identity/1(42).
```

Comment:
Which part is a syntax error? I'm not super familiar with erlang's syntax. This erlang code compiles and runs on my computer.

Reply:
`int_identity(42)`

Reply:
`fun int_identity/1(42).` => `int_identity(42).`

Reply:
Ok, I think [754ab16](https://github.com/gleam-lang/gleam/pull/1664/commits/754ab168d5ceebc47d4b5e1b5a29a7e40d550551) should fix this. Had to add a special match for the case when we are emitting a function call and the function is a `Constant::Var` that contains a `ValueConstructorVariant::ModuleFn` so that this could be handled the same as a normal function call.